BLE Indoor Positioning/src/main/java/com/nexenio/bleindoorpositioning/IndoorPositioning.java

```diff
@@ -30,6 +30,10 @@
     public static final long UPDATE_INTERVAL_MEDIUM = 500;
     public static final long UPDATE_INTERVAL_SLOW = 3000;
 
+    public static final int STRICT_RMS_FILTER = 10;
+    public static final int MEDIUM_RMS_FILTER = 100;
+    public static final int LIGHT_RMS_FILTER = 1000;
```

Comment:
Constants should start with the same prefix, e.g. `RMS_FILTER_MEDIUM`. `THRESHOLD` may be more appropriate than `FILTER`. Also, in a year, I won't be able to associate `RMS` with `Root Mean Square`. That should be documented or renamed.